scripts/superdesk/missing-translations-strings.js

```diff
@@ -25,4 +25,5 @@ gettext('12th update');gettext('13th update');gettext('14th update');gettext('15
 gettext('18th update');gettext('19th update');gettext('20th update');gettext('\'PLACE is a required field\'');
 gettext('Content Profiles');gettext('ANPA Take Key');gettext('Body HTML');gettext('By');gettext('Date');gettext('Located');
 gettext('\'DATELINE is a required field\'');gettext('\"GENRE \'null value not allowed\'');gettext('slugline');gettext('subscription');
+gettext('Populate Abstract');gettext('NTB currency macro');
```

Comment:
Would it make sense to include this file in the superdesk/lang package? It could sit there with all the other translation related stuff.